<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientMainGUI</class>
 <widget class="QMainWindow" name="ClientMainGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="NetworkTab">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <attribute name="title">
        <string>Network</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="lEIP">
             <property name="text">
              <string>localhost</string>
             </property>
             <property name="placeholderText">
              <string>IP</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lEPort">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>8080</string>
             </property>
             <property name="placeholderText">
              <string>Port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bConnect">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="pTERecive"/>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="bLogin">
             <property name="text">
              <string>Login</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bSend">
             <property name="text">
              <string>Send</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lESend"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="BookTab">
       <attribute name="title">
        <string>Book</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QPushButton" name="bSearchBook">
             <property name="text">
              <string>Search Book</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bAddBook">
             <property name="text">
              <string>Add Book</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bEditBook">
             <property name="text">
              <string>Edit Book</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bBorrowBook">
             <property name="text">
              <string>Borrow Book</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bReturnBook">
             <property name="text">
              <string>Return Book</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bGetBook">
             <property name="text">
              <string>Get Book</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEBook"/>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="pTEBookReceive"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="UserTab">
       <attribute name="title">
        <string>User</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QWidget" name="widget_4" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="bAddUser">
             <property name="text">
              <string>Add User</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bChangePassword">
             <property name="text">
              <string>Change Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bGetUser">
             <property name="text">
              <string>Get User</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="bGetBorrowed">
             <property name="text">
              <string>Get Borrowed Books</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lEUser"/>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="pTEUser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
